packages/components/src/dashicon/stories/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { number, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Dashicon from '../';
+
+export default { title: 'Dashicon', component: Dashicon };
+
+export const _default = () => {
+	const icon = text( 'Icon', 'wordpress' );
```

Comment:
Completely unrelated observation to this PR. If we would redo `Dashicon` to use a hash map for icon types, we could reuse it here to show the combo box. It would be 🔥